lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -481,19 +481,7 @@ export class PublicClientApplication {
      *
      */
     public getRedirectUri(): string {
```

Comment:
You could also augment this function to take in the redirect uri from a request:

`return requestRedirectUri || this.config.auth.redirectUri || BrowserUtils.getCurrentUri();`

Reply:
Done